src-frontend/app/adapters/application.js

```diff
@@ -1,7 +1,6 @@
-import DS from 'ember-data';
+import DRFAdapter from './drf';
```

Comment:
We're now using [this](http://dustinfarris.com/ember-django-adapter/) adapter instead of the default `RESTAdapter`.

So now instead of having Ember-friendly JSON responses from Django with vanilla Ember, it's vanilla Django REST Framework and then this DRF-friendly Ember adapter.